app/commands/sre.py

```diff
@@ -47,6 +47,8 @@ def sre_command(ack, command, logger, respond, client, body):
             webhook_helper.handle_webhook_command(args, client, body, respond)
         case "version":
             respond(f"SRE Bot version: {os.environ.get('GIT_SHA', 'unknown')}")
+        case "google-service":
+            google_service.google_service_command(client, body)
```

Comment:
Just so that I understand this, the google_service_command lists the available folders correct (ie runs list-folders)? I assume that this is just for testing and will be removed at a later point?

Reply:
That's also for development purposes so we could make it available only when the dev- prefix is called